apps/challenges/models.py

```diff
@@ -220,3 +220,15 @@ class ChallengeConfiguration(TimeStampedModel):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_zip_configuration'
+
+
+class StarChallenge(TimeStampedModel):
```

Comment:
@taranjeet lets just ignore the feature of storing how many times a challenge was starred/unstarred. This doesn't seem too promising to me.